src/js/current.js

```diff
@@ -12,7 +12,7 @@ RiseVision.ThemeEvents.Current = function() {
 
   function _noEvent() {
     // display message as event name
-    _$current.find(".main--eventName").text("No Event Scheduled");
+    _$current.find(".main--eventName").append("<p class = 'noEventText'>No Event Scheduled<p>");
```

Comment:
@pcsandford Need to avoid creating HTML elements in the JS. A `<span>` is acceptable but anything else should either be directly in the HTML or use a `<template>`. This line should be changed to:
```
_$current.find(".main--eventName p").text(No Event Scheduled);
```
and put the `<p class = 'noEventText'></p>` in the HTML